api/autopilot.go

```diff
@@ -57,9 +57,10 @@ type (
 
 	// HostsConfig contains all hosts settings used in the autopilot.
 	HostsConfig struct {
-		AllowRedundantIPs bool                        `json:"allowRedundantIPs"`
-		MaxDowntimeHours  uint64                      `json:"maxDowntimeHours"`
-		ScoreOverrides    map[types.PublicKey]float64 `json:"scoreOverrides"`
+		AllowRedundantIPs     bool                        `json:"allowRedundantIPs"`
+		MaxDowntimeHours      uint64                      `json:"maxDowntimeHours"`
+		MinRecentScanFailures uint64                      `json:"minRecentScanFailures"`
```

Comment:
That's the persisted autopilot settings right? Won't that cause issues on existing renters since that field will be 0? I don't think I see a default value anywhere that gets set?

Reply:
I wanted to avoid the migration since we're not at 1.0 yet and existing users get a warning in the logs... I added a migration for it.